tests/e2e/cucumber/features/smoke/spaces/publicLink.feature

```diff
@@ -87,9 +87,22 @@ Feature: spaces public link
       """
       Unfortunately, your password is commonly used. please pick a harder-to-guess password for your safety
       """
+    And "Alice" closes the public link password dialog box
     When "Alice" tries to sets the password of the public link named "myPublicLink" of resource "lorem.txt" to "12345678"
     Then "Alice" should see an error message
       """
       Unfortunately, your password is commonly used. please pick a harder-to-guess password for your safety
       """
+    And "Alice" closes the public link password dialog box
+    When "Alice" fills the password of the public link named "myPublicLink" of resource "lorem.txt" to "12345"
+    And "Alice" shows the password of the public link
+    And "Alice" hides the password of the public link
+    And "Alice" closes the public link password dialog box
+    When "Alice" fills the password of the public link named "myPublicLink" of resource "lorem.txt" to "12345"
+    And "Alice" copies the filled password
+    Then "Alice" should get the copied password "12345"
+    And "Alice" closes the public link password dialog box
+    When "Alice" auto fills the password of the public link named "myPublicLink" of resource "lorem.txt"
+    Then "Alice" should see random generated password
```

Comment:
what if we assert the action in the When step itself? we follow this approach in e2e
```suggestion
```